docs/reference/release-notes/6.4.asciidoc

```diff
@@ -70,6 +74,11 @@ coming[6.4.1]
 [float]
 === Known issues
 
+Rollover::
+{es} can't start if any of the shards on the node have been
+<<indices-rollover-index, rolled over>>. (issue: {issue}33316[#33316])
```

Comment:
I think that we need to highlight this more loudly (and the existing known issue for 6.4.0. Please see #32135 for how I think this should be done.